R/pkg/R/RDD.R

```diff
@@ -279,7 +279,7 @@ setMethod("unpersist",
 #' @examples
 #'\dontrun{
 #' sc <- sparkR.init()
-#' setCheckpointDir(sc, "checkpoints")
+#' setCheckpointDir(sc, "checkpoint")
```

Comment:
These two changes are good for consistency although they don't matter to what is ignored by git since this isn't executed right?